### PR TITLE
Link fixer

### PR DESCRIPTION
**Pull request by m4sterbunny**

### CODE REVIEW

File: docs/evm_compatibility/overview.md [hunk 1]
```diff
@@ -44,9 +44,7 @@ The mechanism of gas consumption and calculation of gas fees on Neon EVM differ
 Several Solana-specific differences also impact smart contract development.
 
 ### Upper limit on number of Accounts
-Neon EVM uses [Solana Transaction V2](https://docs.solana.com/proposals/transactions-v2): limiting the maximum number of accounts used in a single transaction to 64. Solana requires that all accounts used in a transaction be specified in order to ensure parallel execution of transactions. 
-
-<!-- todo fix the 404 above -->
+Neon EVM uses Solana Transaction V0: limiting the maximum number of accounts to 32 used in a single transaction unless you extend with [lookup tables](https://docs.solana.com/developing/lookup-tables). Solana requires that all accounts used in a transaction be specified in order to ensure parallel execution of transactions.
```
Comment (m4sterbunny) [inline docs/evm_compatibility/overview.md:47]: @YYYurchenko we had 64 accounts as limit, but the lookup table docs you passed provide 32 -- text updated.
I added the method Solana provides to extend that BUT is that supported by NEVM?

Comment (YYYurchenko) [reply docs/evm_compatibility/overview.md:47]: theoreticly Solana provides 256 accounts ("This "compression" enables storing up to 256 address in a single lookup table ") but in practice we can store only up to 64. Yes, it is already implemented and supported

Comment (m4sterbunny) [reply docs/evm_compatibility/overview.md:47]: Sorry @YYYurchenko I am not making myself clear. The Solana docs state 32 addresses NOT 64
image

Comment (m4sterbunny) [reply docs/evm_compatibility/overview.md:47]: Which means I need confirmation on two counts:
1. Is the upper limit 32 or 64?
2. Are lookup tables supported by Neon EVM?

File: docs/evm_compatibility/overview.md [hunk 2]
```diff
@@ -44,9 +44,7 @@ The mechanism of gas consumption and calculation of gas fees on Neon EVM differ
 Several Solana-specific differences also impact smart contract development.
 
 ### Upper limit on number of Accounts
-Neon EVM uses [Solana Transaction V2](https://docs.solana.com/proposals/transactions-v2): limiting the maximum number of accounts used in a single transaction to 64. Solana requires that all accounts used in a transaction be specified in order to ensure parallel execution of transactions. 
-
-<!-- todo fix the 404 above -->
+Neon EVM uses Solana Transaction V0 with an extension via [account lookup tables](https://docs.solana.com/developing/lookup-tables): this limits the maximum number of accounts to 64 in a single transaction. Solana requires that all accounts used in a transaction be specified in order to ensure parallel execution of transactions.
```
Comment (m4sterbunny) [inline docs/evm_compatibility/overview.md:47]: Pinged my understanding in Slack @YYYurchenko -- this reflects that.

**Review by YYYurchenko — APPROVED** [review]

**Pull request merged**